tracer/src/Datadog.Trace.Tools.Runner/Crank/Importer.cs

```diff
@@ -145,6 +147,14 @@ public static int Process(string jsonFilePath)
                                 {
                                     arch = propItem.Value;
                                 }
+                                else if (propItem.Key == "rate_limit")
```

Comment:
I assume you should use the constant when you read and when you write to avoid mistakes (even though I'm not sure the constant should be in TestTags 🤷‍♂️ )

Reply:
> I'm not sure the constant should be in TestTags

I agree. I think they are custom tags rather than anything "officially" used by the CI Visibility team. Maybe just define them as constants here in `Importer.cs`?

cc @tonyredondo

Reply:
I realized after writing my comment above that Pierre and Andrew had commented basically the same thing elsewhere. Sorry for the duplication 😅

Reply:
Actually, they're two different constants, what is what we write on the crank command-line, which tends to use underscores, the other is what we use as a tag name, which tend to use dashes. I agree, they could be the same value which would probably avoid confusion at some point.

I'll move the constants to this class as suggested.

Reply:
fixed in de8d3d7